lib/smart_answer/calculators/benefit_cap_calculator_data_query.rb

```diff
@@ -1,8 +1,28 @@
 module SmartAnswer::Calculators
   class BenefitCapCalculatorDataQuery
-    attr_reader :data
+    attr_reader :data, :benefits, :questions
     def data
       @data ||= YAML.load_file(Rails.root.join('lib', 'data', 'benefit_cap_data.yml'))
     end
+
+    def benefits
+      @benefits ||=  @data.fetch("benefits").with_indifferent_access
```

Comment:
Is the call to `#with_indifferent_access` important? It be good to test it if so. Actually, I'm not sure there's a test for the `#benefits` method at all, is there?